### PR TITLE
Stable meta-ti updates 

### PR DESCRIPTION
**Pull request by quaresmajose**

### CODE REVIEW

**Comment by ricardosalveti** [comment]
meta-lmp-bsp/recipes-kernel/linux/linux-lmp-ti-staging_git.bb might need to be updated to match linux-ti-staging.

See https://github.com/foundriesio/meta-lmp/commit/6a2421b396d9e0a92e05a0739de0e7c353d09306

**Comment by ricardosalveti** [comment]
Other than that, once done, ask @Tim-Anderson to validate both am62 and am64, I remember we had runtime issues when trying to use a newer bsp.

**Comment by quaresmajose** [comment]
Thanks, I will address your comments and need include some others fixes in meta-lmp that I send upstream.
When everything is place and build I will ask Tim to test.

**Comment by quaresmajose** [comment]
@Tim-Anderson can you please validate the following:
https://ci.foundries.io/projects/lmp/builds/1798/build-am62xx-evm/
https://ci.foundries.io/projects/lmp/builds/1798/build-am64xx-evm/

File: conf/bblayers-bsp.inc [hunk 1]
```diff
@@ -18,6 +18,7 @@ BSPLAYERS = " \
   ${OEROOT}/layers/meta-xilinx-tools \
   ${OEROOT}/layers/meta-tegra \
   ${OEROOT}/layers/meta-ti/meta-ti-bsp \
+  ${OEROOT}/layers/meta-ti/meta-ti-extras \
```
[the remark this answers is not in the document]
Comment (quaresmajose) [reply conf/bblayers-bsp.inc:21]: Sure, we can wait a bit on it. I have sent some patches to remove this layer dependencies in master branch and I can ask to backport to kirkstone but note that the patches taking a rassonable amount of time to be merged.

[the remark this answers is not in the document]
Comment (ricardosalveti) [reply conf/bblayers-bsp.inc:21]: Yup, it is indeed taking longer than desired, but in this case we can probably move the recipe from extras into meta-lmp-bsp (under dynamic layer / meta-arm), as later on we will require this recipe to be functional anyway (we might just need to drop the dependency on ti-paths).

We should also look at migrating this recipe from extras -> bsp upstream.

[the remark this answers is not in the document]
Comment (quaresmajose) [reply conf/bblayers-bsp.inc:21]: I will do this in meta-lmp so we can move forward while the patch is in the queue
https://lists.yoctoproject.org/g/meta-ti/message/15567

[the remark this answers is not in the document]
Comment (quaresmajose) [reply conf/bblayers-bsp.inc]: done in https://github.com/foundriesio/meta-lmp/pull/1008

**Comment by quaresmajose** [comment]
ci-retest

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**